pyspecdata/core.py

```diff
@@ -7413,99 +7369,6 @@ def bootstrap(self,points,swap_out = exp(-1.0),seedval = 10347,minbounds = {},ma
                         recordlist[runno][name] = thiscopy.output(name)
         print(r'\end{verbatim}')
         return recordlist # collect into a single recordlist array
-    def guess(self):
-        r'''provide the guess for our parameters; by default, based on pseudoinverse'''
-        self.has_grad = False
-        if iscomplex(self.data.flatten()[0]):
-            print(lsafen('Warning, taking only real part of fitting data!'))
-        y = real(self.data)
-        # I ended up doing the following, because as it turns out
-        # T1 is a bad fit function, because it has a singularity!
-        # this is probably why it freaks out if I set this to zero
-        # on the other hand, setting a value of one seems to be
-        # bad for very short T1 samples
-        which_starting_guess = 0
-        thisguess = self.starting_guesses[which_starting_guess]
-        numguesssteps = 20
-        #{{{ for some reason (not sure) adding a dimension to y
-        new_y_shape = list(y.shape)
-        new_y_shape.append(1)
-        y = y.reshape(tuple(new_y_shape))
-        #}}}
-        #{{{ evaluate f, fprime and residuals
-        guess_dict = dict(list(zip(self.symbol_list,list(thisguess))))
```

Comment:
this is the type of dictionary I was talking about above

Reply:
yes, I know there was a discussion of a dictionary, I will look into this further